lapack/lapack.go

```diff
@@ -108,21 +108,21 @@ const (
 type SVDJob byte
 
 const (
-	SVDAll       SVDJob = 'A' // Compute all singular vectors
-	SVDInPlace   SVDJob = 'S' // Compute the first singular vectors and store them in provided storage.
-	SVDOverwrite SVDJob = 'O' // Compute the singular vectors and store them in input matrix
-	SVDNone      SVDJob = 'N' // Do not compute singular vectors
+	SVDAll       SVDJob = 'A' // Compute all columns of the matrix U or V.
```

Comment:
Perhaps it is better to avoid referring to specific matrices and use "of the orthogonal matrix" or "on the input matrix" instead.

Reply:
A, U and V make good short hand and are well establish terminology. Perhaps add a note on the type that U and V correspond to the orthogonal matrix and A to the input matrix to make that absolutely clear?

Reply:
I didn't change the comment for the type, I just added the adjective "orthogonal" where U and V are mentioned in the const comments.